config/policy-controller/default/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: kuadrant-
+
+images:
+- name: policy-controller
+  newName: quay.io/kuadrant/policy-controller
+  newTag: main
```

Comment:
images section should be in `config/policy-controller/manager/kustomization.yaml`